index.html

```diff
@@ -111,7 +111,7 @@ <h3 class="project-title">Tonic</h3>
             <li><span>CSS</span></li>
             <li><span>JavaScript</span></li>
           </ul>
-          <a class="btn-see-project" href="#">See Porject</a>
+          <button class="btn btn-see-project" type="button">See Porject</button>
```

Comment:
- [ Optional] As pointed by the previous reviewer, kindly fix the typo from "See Porject" to "See Project" in this and other lines, when you're job hunting after Microverse your portfolio is gonna be one of the first stops a recruiter will do when looking if you should be asked for an interview, it's very important for it to be as professional as possible

Reply:
thank you very much, @leolpaz! will do it right now.